overrides/kubejs/server_scripts/unify_barks.js

```diff
@@ -0,0 +1,40 @@
+ ServerEvents.tags('item', event => {
+//unify tags
+    event.add('forge:barks', '#immersive_weathering:bark')

```

Comment:
If this is not a natively forge tag, i think it would be preferable to put under our own namespace.
additionally, you can use an array instead of invoking the add function twice
```suggestion
    event.add('mce2:barks', ['#immersive_weathering:bark', '#farmersdelight:barks'])

```

Reply:
No worries, was a placeholder namespace until review. Was unsure on what to use, thanks for the clarity!